package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "eoslime-shape-react",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Write a simple description of what it is about

Reply:
added description